site/scripts/create-db-indexes.ts

```diff
@@ -28,6 +28,12 @@ const script = async () => {
       .createIndex({ email: 1 }, { unique: true, sparse: true });
   });
 
+  await catchAndLog(async () => {
+    await db
+      .collection<UserDocument>(User.COLLECTION_NAME)
+      .createIndex({ shortname: 1 });
```

Comment:
Should the shortnames be unique as well?

Reply:
Oops, yes - thought I had. Fixed